_data/themes/beach.yml

```diff
@@ -0,0 +1,47 @@
+# should we name these colors based on what elements they change or keep them the same name as the uswds variable colors?
+
+colors:
+# content link color, hover color in nav, active highlight in nav, primary button bg color, return to to link color
+# usa-banner link color ???
+  primary: 'red'
+# content link hover color, usa-banner link hover color, primary button hover bg color, return to top link hover color
```

Comment:
Some of these colors also change the the USA banner, `primary`, `primary-darker` and `base` (which I didn't expect). 
I think we need to account for the USA banner having it's own styles.